solution-base/deps.txt

```diff
@@ -1,10 +1,6 @@
-kubedb/operator:v0.13.0-rc.0
-kubedb/redis:5.0.3-v1
-kubedb/redis_exporter:v0.21.1
-kubedb/mongo:4.0-v1
-kubedb/mongo-tools:4.0-v1
-kubedb/busybox:latest
-kubedb/mongodb_exporter:v1.0.0
```

Comment:
Since these images are removed, shouldn't the KubeDB manifests be removed as well? Not that it would hurt keeping them around :slightly_smiling_face:

Reply:
@gdemonet I had left the manifests in order to delete resource with `kubectl delete -f`

Reply:
Makes sense!